doc/themes/scikit-learn-modern/nav.html

```diff
@@ -0,0 +1,52 @@
+{%- if pagename != 'index' and pagename != 'documentation' %}
+  {%- set nav_bar_class = "sk-docs-navbar" %}
+  {%- set top_container_cls = "sk-docs-container" %}
+{%- else %}
+  {%- set nav_bar_class = "sk-landing-navbar" %}
+  {%- set top_container_cls = "sk-landing-container" %}
+{%- endif %}
+
+<nav id="navbar" class="{{ nav_bar_class }} navbar navbar-expand-md navbar-light bg-light py-0">
+  <div class="container-fluid {{ top_container_cls }}">
+    {%- if logo %}
+      <a class="navbar-brand" href="{{ pathto('index') }}">
+        <img
+          src="{{ pathto('_static/' + logo, 1) }}"
+          height="38"
+          alt="logo"/>
+      </a>
+    {%- endif %}
+    <button
+      class="navbar-toggler"
+      type="button"
+      data-toggle="collapse"
+      data-target="#navbarSupportedContent"
+      aria-controls="navbarSupportedContent"
+      aria-expanded="false"
+      aria-label="Toggle navigation"
+    >
+      <span class="navbar-toggler-icon"></span>
+    </button>
+
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav mr-auto">
+        <li class="nav-item">
+          <a class="nav-link" href="{{ pathto('index') }}">Home</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="{{ pathto('install') }}">Install</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="{{ pathto('documentation') }}">Documentation</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="{{ pathto('auto_examples/index') }}">Examples</a>
+        </li>
+      </ul>
+      <div class="sk-search-form">
+          <div class="gcse-search" id="cse" data-linktarget="_parent"></div>
+      </div>
+      </form>
```

Comment:
stray closing tag. (Found by https://validator.w3.org/nu/?doc=https%3A%2F%2Fthomasjpfan.github.io%2Fscikit-learn-website%2Fdocumentation.html)

Reply:
Ah yes, thank you for catching this! This was left over from a custom search bar that was converted into a google search bar.